language/move-analyzer/editors/code/src/context.ts

```diff
@@ -92,5 +106,16 @@ export class Context {
         log.info('Starting client...');
         const disposable = client.start();
         this.extensionContext.subscriptions.push(disposable);
+        this._client = client;
+        await this._client.onReady();
+    }
+
+    /**
+     * Returns the client that this extension interacts with.
+     *
+     * @returns lc.LanguageClient
+     */
+    getClient(): lc.LanguageClient | undefined {
```

Comment:
unless replacing the client after constructing is required, this could be replaced with a [readonly property](https://www.typescriptlang.org/docs/handbook/2/objects.html#readonly-properties):
`private _client` becomes `readonly client`